app/Commands/CheckDockerComposeVersion.php

```diff
@@ -9,7 +9,7 @@
 
 class CheckDockerComposeVersion extends Command
 {
-    const DOCKER_COMPOSE_MIN_VERSION = '1.24';
+    const DOCKER_COMPOSE_MIN_VERSION = '1.23';
```

Comment:
Docker for Mac stable version is `1.23`, so i downgraded.

Another thing about this is that we're checking the wrong param,  the thing we really wanted to check is `docker version --format '{{.Server.APIVersion}}'`, but as we are constraining to docker `18.09` i think is already guarantees having the APIVersion we need

Reply:
Guess we may want to check both (not replace the checked value, but add a check for the APIVersion as well.